packages/rest/src/rest.server.ts

```diff
@@ -156,7 +158,7 @@ export class RestServer extends Context implements Server, HttpServerLike {
   ) {
     super(app);
 
-    options = options || {};
+    options = options || ({} as RestServerConfig);
```

Comment:
This type cast should not be necessary, please check if it can be removed after `HttpOptions#protocol` was changed to an optional property per my earlier comment.